src/dirac_cwl_proto/commands/generic/jobTypes.py

```diff
@@ -0,0 +1,8 @@
+from dirac_cwl_proto.commands import JobProcessorBase
+
+from .modules import DownloadConfig, GroupOutputs
+
+
+class TestingJobType(JobProcessorBase):
+    preprocess_commands = [DownloadConfig]
+    postprocess_commands = [GroupOutputs]
```

Comment:
I think this is the right approach, but ideally we should move these into https://github.com/aldbr/dirac-cwl-proto/blob/5ed828cc018d87151e407f3c4fa43d62f18cf206/src/dirac_cwl_proto/execution_hooks/plugins/core.py#L19